### PR TITLE
Added the bare bones of ISO creation.

### PR DESCRIPTION
**Pull request by Arignir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+
+The iso can be burned on an external device (like an USB flash drive or an optical disk) using `dd`:
+
+```bash
+dd if=/path/to/raven-os.iso of=/dev/sdx
+```
+
+Where `sdx` is the device you want to burn the iso on.
+
+The computer can be booted from the selected device by tweaking some BIOS/UEFI settings, commonly reffered as the boot order. You want to make sure the device will be the first loaded, before the main hard drive. See your motherboard's manual for more details.
```
Comment (doom) [inline README.md]: `referred to as`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+When the Raven-OS menu appears, select the "Live" if you want to try Raven-OS, or "Graphical Install" or "Textual Install" if you want to install Raven-OS on your system.
```
Comment (doom) [inline README.md]: `the "Live" entry if`

**Review by doom — COMMENTED** [review]
Some typos should be fixed, otherwise the script looks good to me

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+```bash
+root $ ./iso.sh
+```
+
+If the script succeeds, the message `Done. The iso is located at "/path/to/raven-os.iso".` is displayed.
+
+
+## Trying Raven-OS on real hardware
+
+The iso can be burned on an external device (like an USB flash drive or an optical disk) using `dd`:
```
Comment (melis-m) [inline README.md]: it's `a USB` and not `an USB`

File: iso.sh [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    generate_grub_cfg "PARTUUID=$partuuid" >> "$mnt_linux/boot/grub/grub.cfg"
+
+    # Unmount and remove the loopback devices
+    umount -R "$mnt_linux"
+    umount -R "$mnt_efi"
+    losetup -d "$device"
+
+    sync
+
+    # Move the iso to it's final destination
```
Comment (melis-m) [inline iso.sh]: Should be `its`, not `it's`

**Review by melis-m — DISMISSED** [review]

File: iso.sh (hunk 4 above)
Comment (Arignir) [reply iso.sh]: I make this error all the time :( Thanks!

**Review by Pedrochu — APPROVED** [review]

**Pull request merged**